Framework/Core/src/SendingPolicy.cxx

```diff
@@ -20,8 +21,12 @@ namespace o2::framework
 std::vector<SendingPolicy> SendingPolicy::createDefaultPolicies()
 {
   return {SendingPolicy{
-    .name = "default",
-    .matcher = [](DeviceSpec const& spec, ConfigContext const& ctx) { return true; },
-    .send = [](FairMQDevice& device, FairMQParts& parts, std::string const& channel) { device.Send(parts, channel, 0); }}};
+            .name = "dispatcher",
+            .matcher = [](DeviceSpec const& spec, ConfigContext const& ctx) { return spec.name == "Dispatcher"; },
```

Comment:
Instead of this, could you please check if a label "Dispatcher" exists? In the future we might have to add some hashes in the name to allow for multiple Dispatchers.

Reply:
I guess something like this?

Reply:
Yes, thank you